mujoco_ros/include/mujoco_ros/defs.h

```diff
@@ -6,4 +8,13 @@
 #include <strstream>
 #include <iomanip>
 #include "GLFW/glfw3.h"
-#include "ros/ros.h"
\ No newline at end of file
+#include "ros/ros.h"
+#include "mujoco/mujoco.h"
+
+namespace mj_ros
+{
+namespace constants
+{
+static double frequency{ 50.0 };
```

Comment:
As we discussed offline, maybe this should not be a constant and set instead as `1.0/m->opt.timestep`.